pkg/destinations/redshift/redshift.go

```diff
@@ -0,0 +1,67 @@
+package redshift
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	_ "github.com/lib/pq"
+	"github.com/scratchdata/scratchdata/util"
+)
+
+type RedshiftServer struct {
```

Comment:
This is missing the `delete_from_s3` setting, see Step 2 of #117

Reply:
delete_from_s3 will be from config file too?

Reply:
Yes. Look at #117. Under "Implement Data Insertion":

> If the delete_from_s3 flag is true, then delete the file from S3 if the copy command has been successful.